x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_notify_when.tsx

```diff
@@ -49,7 +49,7 @@ export const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<RuleNotifyWhenType>
         <EuiText size="s" color="subdued">
           <p>
             <FormattedMessage
-              defaultMessage="Actions run when the alert status changes."
+              defaultMessage="Actions repeat if the alert status changes."
```

Comment:
I don't know if `repeat` is accurate here. As written above, an action would be triggered when moving between statuses, but I wouldn't consider this repeating.

Reply:
Thanks, changed!